assets/js/modules/tagmanager/components/common/index.js

```diff
@@ -29,3 +29,4 @@ export { default as FormInstructions } from './FormInstructions';
 export { default as UseSnippetSwitch } from './UseSnippetSwitch';
```

Comment:
Nit: exports here are alpha sorted to make it easier to scan.

```suggestion
export { default as TagCheckProgress } from './TagCheckProgress';
export { default as UseSnippetSwitch } from './UseSnippetSwitch';
```